client/my-sites/site-settings/settings-newsletter/ReplyToSetting.tsx

```diff
@@ -0,0 +1,61 @@
+import { FormLabel } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormRadio from 'calypso/components/forms/form-radio';
+
+type ReplyToSettingProps = {
+	value?: string;
+	disabled?: boolean;
+	updateFields?: ( fields: { [ key: string ]: unknown } ) => void;
+	isWPcomSite?: boolean;
+};
+
+export const ReplyToSetting = ( {
+	value = 'no-reply',
+	disabled,
+	updateFields,
+	isWPcomSite,
+}: ReplyToSettingProps ) => {
+	const translate = useTranslate();
+	return (
+		<FormFieldset>
+			<FormLabel className="increase-margin-bottom-fix">
+				{ translate( 'Reply-to settings' ) }
+			</FormLabel>
+			<p>
+				{ translate( 'Choose who receives emails when subscribers reply to your newsletter.' ) }
+			</p>
+			<FormLabel>
+				<FormRadio
+					value="no-reply"
+					checked={ value === 'no-reply' || value === '' }
+					onChange={ () => updateFields?.( { jetpack_subscriptions_reply_to: 'no-reply' } ) }
+					disabled={ disabled }
+					label={ translate( 'Replies are not allowed.' ) }
```

Comment:
It seems we are not using trailing dots for the options labels anywhere else on this page. Let's remove them?

Reply:
Thanks!